test/strings/search.jl

```diff
@@ -408,3 +408,13 @@ for T = (UInt, BigInt)
         @test findprev(isletter, astr, T(x)) isa Int
     end
 end
+
+# issue 37280
```

Comment:
Better create a `@testset` and give it an explicit name (this is a feature, not just a bug). Also use a loop to test both `Int8` and `UInt8`, including mixing the two in each argument.